introduction.md

```diff
@@ -0,0 +1 @@
+# Hello! I'm gus07ven and I'm learning GitHub. Thank you for your help!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.